Move how to add ADB to PATH (#130)
Coppersource/gm0: source/docs/software/using-android-studio.rst

.. note:: You can use logcat via ADB with the :code:`adb logcat` command. This is extremely useful for debugging as it allows you to look at the logs wirelessly which saves time. Remember, logcat is the *best* way to debug your software.

Add ADB To PATH
^^^^^^^^^^^^^^^

Adding variables to PATH:

- `Windows <https://docs.alfresco.com/4.2/tasks/fot-addpath.html>`_

> - `Unix/Linux <https://unix.stackexchange.com/questions/26047/how-to-correctly-add-a-path-to-path>`_

Reviewer: macOS?

> - `Windows <https://docs.alfresco.com/4.2/tasks/fot-addpath.html>`_
> - `Unix/Linux <https://unix.stackexchange.com/questions/26047/how-to-correctly-add-a-path-to-path>`_

Reviewer: ```suggestion
- `Windows <https://docs.alfresco.com/4.2/tasks/fot-addpath.html>`_
- `Linux/Unix (bash) <https://unix.stackexchange.com/questions/26047/how-to-correctly-add-a-path-to-path>`_
- `macOS (zsh) <https://koenwoortman.com/zsh-add-directory-to-path/>`_
```

> If you want to use ADB anywhere, you can set a system variable for it. Simply follow an online tutorial for adding to PATH and set the PATH to the :code:`platform-tools` directory. Once you do that, you can run ADB commands from anywhere on your system.

Reviewer: ```suggestion
If you want to use ADB from any directory, add it to PATH. Follow an online tutorial for adding to PATH and set the PATH to the :code:`platform-tools` directory. Once you do that, you can run ADB commands from anywhere on your system.
```

Connecting to a Phone Wirelessly
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

#. Plug the robot controller phone into your computer.
#. Run the command :code:`adb devices` in the :code:`platform-tools` directory and see if the phone shows up.
#. Run :code:`adb usb` and then :code:`adb tcpip 5555`. You can then unplug the phone.

> #. Connect to the same WiFi network the device is either hosting or on. The WiFi direct network created by the phone should be called "TEAMNUMBER-RC" or some small derivation of that. It may include extra letters if you have multiple devices per team. Refer to RS01 in `Game Manual 1 <https://www.firstinspires.org/sites/default/files/uploads/resource_library/ftc/game-manual-part-1-traditional-events.pdf>`_ for further details on the network naming scheme.

Reviewer: ```suggestion
#. Connect to the same WiFi network the device is either hosting or on. The WiFi direct network created by the phone should be called "TEAMNUMBER-RC" or some small derivation of that. It may include extra letters if you have multiple devices per team. Refer to RS01 in |gm1| for further details on the network naming scheme.
```

|gm1| is replaced by Game Manual Part 1; it'd be nice if it was replaced with Game Manual Part 1 with a link, but this way that can be done globally easily. I'll make it include a link soon.

#. Connect to the phone using :code:`adb connect 192.168.49.1:5555`. If this doesn't work, recheck the IP address of the phone and try again with that IP address if it is different.

Connecting to a Control Hub Wirelessly
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

> #. Connect to the WiFi hotspot hosted by the Control Hub. The hotspot should be called "TEAMNUMBER-RC" or some small derivation of that. It may include extra letters if you have multiple devices per team. Refer to RS01 in `Game Manual 1 <https://www.firstinspires.org/sites/default/files/uploads/resource_library/ftc/game-manual-part-1-traditional-events.pdf>`_ for further details on the network naming scheme.

Reviewer: ```suggestion
#. Connect to the WiFi hotspot hosted by the Control Hub. The hotspot should be called "TEAMNUMBER-RC" or some small derivation of that. It may include extra letters if you have multiple devices per team. Refer to RS01 in |gm1| for further details on the network naming scheme.
```
